.github/workflows/docs-latest-public.yaml

```diff
@@ -1,9 +1,7 @@
 name: Docs - Build latest docs and deploy to GitHub pages on main
 
 on:
-  push:
```

Comment:
Hmm... I think we want to eventually use build artifacts for the docs here too.

I think for now this ok so we don't make this PR bigger than it needs to be. But we should probably file an issue to make sure the above is work is captured for the future